backend/src/db/migrations/20200312140328-bulk_upload_to_s3.js

```diff
@@ -32,39 +32,38 @@ export async function up(next) {
   try {
     // Implement your migration here.
     const { records } = await transaction.run('MATCH (image:Image) RETURN image.url as url')
-    let urls = records.map(r => r.get('url'))
-    urls = urls.filter(url => url.startsWith('/uploads'))
+    let urls = records.map((r) => r.get('url'))
+    urls = urls.filter((url) => url.startsWith('/uploads'))
     const locations = await Promise.all(
       urls
-        .map(url => {
+        .map((url) => {
           return async () => {
             const { pathname } = new URL(url, 'http://example.org')
             const fileLocation = path.join(__dirname, `../../../public/${pathname}`)
             const s3Location = `original${pathname}`
```

Comment:
I see that the images now live on our community staging. Do we want to have them in a folder `uploads/...`? I would say, we should omit that folder name, no?

Reply:
Or, maybe not. We can clearly distinguish the images that have been uploaded with this migration. I case we have to do some disaster recovery, this could come in handy.

Reply:
the reason it lives there is because it's part of the `Image.url`
yeah, agree with the second part of your message, let's KISS